packages/extension/src/newtab/MainFeedPage.tsx

```diff
@@ -29,6 +25,10 @@ const PostsSearch = dynamic(
     ),
 );
 
+const DndModal = dynamic(
+  () => import(/* webpackChunkName: "search" */ './DndModal'),
```

Comment:
```suggestion
  () => import(/* webpackChunkName: "dnd" */ './DndModal'),
```
The webpack chunk name sets the bundle name of this dynamic import. We don't want to merge it with an existing bundle.